website/source/docs/providers/aws/r/cloudwatch_metric_alarm.html.markdown

```diff
@@ -67,7 +67,7 @@ The following arguments are supported:
 * `namespace` - (Required) The namespace for the alarm's associated metric. See docs for the [list of namespaces](https://docs.aws.amazon.com/AmazonCloudWatch/latest/DeveloperGuide/aws-namespaces.html).
   See docs for [supported metrics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/DeveloperGuide/CW_Support_For_AWS.html).
 * `period` - (Required) The period in seconds over which the specified `statistic` is applied.
-* `statistic` - (Required) The statistic to apply to the alarm's associated metric.
+* `statistic` - (Optional) The statistic to apply to the alarm's associated metric.
```

Comment:
Add in documentation here specifying that you can only have one of `statistic` or `extended_statistic`